e2e/tests/components/date-time-picker/date-time-picker.e2e-spec.ts

```diff
@@ -200,4 +200,20 @@ describe('Date Time Picker Tests', () => {
         }
 
     });
+
+    it('should correctly set the date when select an adjacent month', async () => {
+
+        const date = await page.getDate('31');
+        await date.click();
+
+        // check if it now has the active class
+        const classes = await date.getAttribute('class');
+        expect(classes.indexOf('active')).toBeGreaterThan(-1);
```

Comment:
This should be after the `page.nextDate.click()` line, where I put the previous comment.